blazor/datagrid/column-menu.md

```diff
@@ -147,6 +147,8 @@ else if (OrderIDFreezeDirection == FreezeDirection.Right)
 
 ```
 
+![Customize Column Menu in Blazor DataGrid](images/blazor-datagrid-custom-column-menu.gif)
+
 <!-- Column menu events
 
 The grid component triggers the below events when column menu operations are performed,
```

Comment:
The grid component triggers the events below when column menu operations are performed: